astropy/io/ascii/rst.py

```diff
@@ -0,0 +1,54 @@
+# Licensed under a 3-clause BSD style license
+"""
+:Author: Simon Gibbons (simongibbons@gmail.com)
+"""
+
+from __future__ import absolute_import, division, print_function
+
+from .core import DefaultSplitter
+from .fixedwidth import (FixedWidth,
+                         FixedWidthData,
+                         FixedWidthHeader,
+                         FixedWidthTwoLineDataSplitter)
+
+class SimpleRSTHeaderSplitter(DefaultSplitter):
+    delimiter = ' '
+
+class SimpleRSTHeader(FixedWidthHeader):
+    position_line = 0
+    start_line = 1
+    splitter_class = SimpleRSTHeaderSplitter
+    position_char = '='
+
+class SimpleRSTDataSplitter(FixedWidthTwoLineDataSplitter):
```

Comment:
Yes you are right, It's cleaner IMO to instead modify `get_fixedwidth_params` in `SimpleRSTHeader` to deal with the unbounded right hand column. This also has the benefit of allowing the right hand column name to be unbounded also (to fit with the spec)